app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -71,7 +71,7 @@ public void onPos(final SourceContent sourceContent, boolean b) {
                         if (imageList == null || imageList.size() == 0) {
                             holder.previewImageView.setVisibility(View.GONE);
                         } else {
-                            Glide.with(context).load(imageList.get(0))
+                            Glide.with(context.getApplicationContext()).load(imageList.get(0))
```

Comment:
I searched for cause. I found that use getApplicationContext. It solve the problem